pyigm/cgm/utils.py

```diff
@@ -40,6 +45,20 @@ def calc_rho(galaxy, igm_sys, cosmo, ang_sep=None, correct_lowz=True):
             rhos.append(irho.value)
             angs.append(iang)
         return np.array(rhos)*u.kpc, angs
+    # Galactic?
+    if Galactic:
+        d_Sun = 8.0 * u.kpc
```

Comment:
Perhaps 'd_Sun = 8.0 * u.kpc' should be a default argument of calc_rho() instead of hard-coded?  People are still refining this measurement.